data/extra.csv

```diff
@@ -20,9 +20,9 @@ Hope's Scarf (XIII),Accessory,4,,15,,,,,,,15,,,,,
 Inferno Fang (VIII),Dagger,5,64,,95,,,,,113,,95,,,,
 Cat's Claws (IX),Fist,5,77,,95,,,,,126,,95,,,,
 Heavy Coat (XII),Light Armor,3,,,,20,21,99,,,,,48,48,99,
-Witch's Hat (XIII),Hat,3,,18,,36,70,,15,,24,,51,100,,21
+Witch's Hat (XIII),Hat,5,,18,,36,70,,15,,24,,51,100,,21
```

Comment:
Are you sure about these?  The stats I used here do not match the ones from gamematome.jp, but these numbers result in the right stats showing up in-game.

Or, that thing I just said, but I did it wrong, and you are right - please double check against in game.